pkgs/misc/drivers/epkowa/default.nix

```diff
@@ -30,7 +30,15 @@ let plugins = {
     version = "2.30.4";
 
     src = fetchurl {
-      url = "https://download2.ebz.epson.net/iscan/plugin/perfection-v330/rpm/x64/iscan-perfection-v330-bundle-${version}.x64.rpm.tar.gz";
+      # To find new versions, visit
+      # http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX and search for
+      # some printer like for instance "WF-7210" to get to the most recent
+      # version.  
+      # NOTE: Don't forget to update the webarchive link too!
+      url = [
```

Comment:
It seems `url` should be plural.

Reply:
Done. Also Replaced all the urls there. Unfortunately, I wasn't able to trigger all of the urls to be cached by webarchive. I guess my ip did hit some limit. Maybe you can trie to get all the files from epkowa cached, by accessing the links and clicking on "save
```
sed 's_\(\s*\)url = "\(.*\)";_\1urls = [\n\1  "\2"\n\1  "https://web.archive.org/web/\2"\n\1];_g' --in-place *.nix
``` 
Did the job